docs/StardustDocs/topics/ColumnSelectors.md

```diff
@@ -25,17 +25,17 @@ df.move { name.firstName and name.lastName }.after { city }
 
 **Definitions**
 
-<dataFrame src="kdocs/org.jetbrains.kotlinx.dataframe.api.ColumnsSelectionDsl.DslGrammar.DefinitionsPartOfGrammar.html"/>
+<inline-frame src="resources/org.jetbrains.kotlinx.dataframe.api.ColumnsSelectionDsl.DslGrammar.DefinitionsPartOfGrammar.html" width="100%"/>
```

Comment:
btw, they don't appear in the `resources` folder by gradle, right? I thought they appeared in `snipptets`